tests/lib/constants/constants.py

```diff
@@ -302,3 +302,10 @@ def __init__(self, code, status, title, detail, links={}):
     "Too many requests",
     "This endpoint is currently receiving a high volume of requests and is being rate limited."
 )
+
+ERROR_CANNOT_SET_CONTACT_DETAIL = Error(
```

Comment:
```suggestion
ERROR_CANNOT_SET_CONTACT_DETAILS = Error(
```
Super minor, but bothers my OCD :-)